examples/tutorials/hello-gltf/app.ts

```diff
@@ -58,7 +58,18 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
         u_NormalMatrix: new Matrix4(worldMatrix).invert().transpose()
       })
 
-      model.updateModuleSettings({lightSources});
+      model.shaderInputs.setProps({
+        lights: {lightSources}
```

Comment:
The source of the issue is likely the use of the `lights` module. Instead we should use the `lighting` module, by switching out the dependency in the `pbr` shader module